Documentation/design-docs/host-startup-hook.md

```diff
@@ -51,13 +51,14 @@ will be inherited by child processes by default. It is up to the
 desired.
 
 Specifically, hostpolicy starts up coreclr and sets up a new
-AppDomain. It then invokes a private method in
-`System.Private.CoreLib`, which will call each
+AppDomain, passing in the startup hook variable if it was
```

Comment:
Can you add the exact key name here?  https://github.com/dotnet/core-setup/blob/7e88484b2ab88b98088c2f40b620c7593b0fdd16/src/corehost/cli/hostpolicy.cpp#L97